wsi/Android.mk

```diff
@@ -64,6 +64,7 @@ endif
 LOCAL_CPPFLAGS += \
         -DHWC2_INCLUDE_STRINGIFICATION \
         -DHWC2_USE_CPP11 \
+        -DUSE_DRM_CTM \
```

Comment:
Should we have a BOARD Config to disable this ?
i.e ifneq ($(strip $(BOARD_DISABLE_DRM_CTM)), true)
LOCAL_CPPFLAGS += -DUSE_DRM_CTM
endif